frontend/src/ui/modules/components/navbar.jsx

```diff
@@ -13,12 +12,15 @@ import Tooltip from '@mui/material/Tooltip';
 import Settings from '@mui/icons-material/Settings';
 import Logout from '@mui/icons-material/Logout';
 import { setAuthHeader, setUser } from '../../../helpers/axios_helper';
+import { stringAvatar } from '../../../helpers/avatar_helper'
 
 
 function Navbar() {
   const handleLogout = () => {
     setAuthHeader(null);
     setUser(null);
```

Comment:
I think these(20-21) won't be neccessary anymore then since we have localstorage.clear()